tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,122 @@
+app-id: tv.plex.PlexHTPC
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Plex
+finish-args:
+  - --socket=x11
+  # currently not working - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-cache:rw
+  - --filesystem=xdg-config:rw
```

Comment:
why do you need to access other app (non flatpak) configuration or worse write into it? same for `xdg-data` and `xdg-cache`

Reply:
Yeah I don't actually need that, I just misunderstood what these permissions do. Thanks for your help! 🙏